src/framework/preferences/html/PrefsEditorTemplate-lineSpace.html

```diff
@@ -1,5 +1,6 @@
 <div class="flc-prefsEditor-line-space">
     <h2><span class="fl-icon-line-space"></span><label class="flc-prefsEditor-line-space-label" for="line-space"></label></h2>
```

Comment:
Currently the description of [FLUID-5678](https://issues.fluidproject.org/browse/FLUID-5678) discusses addressing three things.

- Detailed description/instruction text for function of tools e.g. change size of site text  
- Use of labels on the headings is inconsistent – some headings are labels for a panel’s control and some are not. Input identification/labeling should be consistent   
- Wording for measure of scale with text size and line height as x Times may be confusing 

If you'd like to do the labels and etc. in other pull requests, that's okay, but please create new jiras for them and update the description for this one.